src/big/big_float.cr

```diff
@@ -393,6 +393,43 @@ class String
 end
 
 module Math
+  # Returns the unbiased base 2 exponent of the given floating-point *value*.
+  def ilogb(value : BigFloat) : Int64
+    LibGMP.mpf_get_d_2exp(out exp, value)
+    (exp - 1).to_i64
+  end
+
+  # Returns the unbiased radix-independent exponent of the given floating-point *value*.
+  #
+  # For `BigFloat` this is equivalent to `ilogb`.
+  def logb(value : BigFloat) : BigFloat
+    LibGMP.mpf_get_d_2exp(out exp, value)
+    (exp - 1).to_big_f
```

Comment:
Why not just make this call the other method when it's equivalent?
```suggestion
    ilogb(value)
```

Reply:
Their return types are different (I don't know why C does the same, to be honest).

Reply:
If `(exp - 1).to_i64.to_big_f` never leads to a loss of information then `ilogb(value).to_big_f` is okay.